demo/index.html

```diff
@@ -16,15 +16,19 @@
     <meta name="mobile-web-app-capable" content="yes">
     <meta name="apple-mobile-web-app-capable" content="yes">
 
-    <script src="../../../@webcomponents/webcomponentsjs/webcomponents-bundle.js"></script>
-    <script type="module" src="../../paper-styles/typography.js"></script>
+    <script src="https://unpkg.com/@webcomponents/webcomponentsjs@^2.0.0/webcomponents-loader.js"></script>
+
+    <!-- You need to import this library first for the behaviour to work -->
+    <script src="../node_modules/intl-messageformat/dist/intl-messageformat.js"></script>
```

Comment:
Should this come from unpkg.com also? (BTW, the readme also shows this library being loaded from `node_modules`, while others are loaded from unpkg.)

Reply:
this is a direct dependency of this element, not a polyfill or something extraneous (like, the whole point of this element is to wrap this llibrary. it weirds me out that it would not be served from the same place, i.e. locally, which would make this element unusable offline)